data/projects/datacapstats.yaml

```diff
@@ -0,0 +1,4 @@
+name: datacapstats
```

Comment:
Project name should be either:
`filplus-dashboard-webapp-fidlabs`
or simply `fidlabs` (if you have other relevant repos)